src/style.css

```diff
@@ -1 +1,78 @@
 /* Styles go here */
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap");
+
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  --color-link-active: #00acdc;
+  --color-link: #000;
```

Comment:
There is a typo in the variable name 'color-backgtround-header'. It should be 'color-background-header'.